src/lib/PostgresMetaRoles.ts

```diff
@@ -51,10 +51,12 @@ WHERE
     if (offset) {
       sql += ` OFFSET ${offset}`
     }
-    const result = await this.query(sql)
+    const result = (await this.query(sql)) as PostgresMetaResult<PostgresRole[]>
     if (result.data) {
-      result.data = result.data.map((role: any) => {
+      result.data = result.data.map((role) => {
+        // @ts-ignore
         role.config = changeRoleConfig2Object(role.config)
+        role.is_default_role = role.name in DEFAULT_ROLES
```

Comment:
I believe you need to use `includes` here.

DEFAULT_ROLES is a string array

![Screenshot 2023-01-28 at 12 06 13](https://user-images.githubusercontent.com/14073399/215263230-c37619eb-ee58-4d0e-af6e-d22c7472b95d.png)

Reply:
`in` would work for checking whether an object has a property

Reply:
Thanks. I always forget about this in Javascript.
Some confusion between Javascript and Python